README.md

```diff
@@ -6,21 +6,33 @@ This repository contains three functional MRI tasks implemented as *Psychopy3* e
 * Quality-control task (QCT): [`task-qct_bold.psyexp`](https://github.com/TheAxonLab/HCPh-fMRI-tasks/blob/master/task-qct_bold.psyexp)
 * Resting-state fMRI (rest): [`task-rest_bold.psyexp`](https://github.com/TheAxonLab/HCPh-fMRI-tasks/blob/master/task-rest_bold.psyexp)
 
-## Positive control task
+## Quality control task
 The task contains a visual trial block, an eye-movement trial block, and a finger-tapping trial block.
-The task was originally derived from «Version A» of the task proposed by Harvey et al.:
+The task was adapted from «Version A» of the task proposed by Harvey et al.:
 
      Harvey J, Demetriou L, McGonigle J, Wall MB. 2018. A short, robust brain activation
      control task optimised for pharmacological fMRI studies. PeerJ 6:e5540
      doi:https://doi.org/10.7717/peerj.5540
 
+The original implementation of their task is found at [mattwall1103/fMRI-Control-Task](https://github.com/mattwall1103/fMRI-Control-Task). Please cite their paper whenever you use their code.
+
 The most prominent changes to the original task are:
 
 - Elimination of the auditive block (since we do not plan the parcipant to wear headphones)
 - Substitution of the motor task (button pressing) with a simple finger-tapping paradigm
 - Enabling eye-tracking with our SR instruments device.
 
-The original implementation of their task is found at [mattwall1103/fMRI-Control-Task](https://github.com/mattwall1103/fMRI-Control-Task). Please cite their paper whenever you use their code.
+### Detailed description of QCT
+
+The QCT consists of four paradigms whose presentation order and realization is randomly selected. The background for all tasks is grey (HEX #808080). The units reported here correspond to the normalised unit.
```

Comment:
```suggestion
The QCT consists of four paradigms whose presentation order and realization are randomly selected.
The background for all tasks is black (HEX #000000).
The units reported here correspond are *Psychopy*'s "normalized units".
```

Reply:
@oesteban The background is black, really?

Reply:
Yup, so much better in many ways

Reply:
Ok who knew. I have to fetch the updates of the code and verify what I wrote then.

Reply:
@oesteban Ok it turns out I had the latest version of the task. Thus, can you double-check that the stimuli's colors and appearance correspond to what you see in the scanner?
I can't run the task, so I was trying to infer info from the psychopy design, but apparently, I was not so successful.

Reply:
I can tell you what I see during the session by heart. And for sure the background is black :D

I'll check because the experiments we are using could have some very minimal changes. We should add a git tag with the version we are using. I can take that issue.

Reply:
> I can't run the task, so I was trying to infer info from the psychopy design, but apparently, I was not so successful.

Our SOPs are there exactly to avoid this. I think it would be important for you to achieve a working psychopy installation on your laptop. Not because of testing the task (that too), but because we want to be sure others can replicate our settings.

I am 99% convinced that if you follow the SOPs to the letter (from Ubuntu version and psychopy installation steps all the way to how to simulate the MMBT-S with `socat` and deploying our trigger forward service), you should be able to run the four tasks.